libs/blocks/commerce/commerce.js

```diff
@@ -0,0 +1,152 @@
+import { createTag, getConfig, loadScript } from '../../utils/utils.js';
+import { getTacocatEnv, runTacocat, buildCheckoutButton, getCheckoutContext, omitNullValues } from '../merch/merch.js';
+
+window.tacocat.loadPromise = new Promise((resolve) => {
+  const { env, locale } = getConfig();
+  const {
+    literalScriptUrl,
+    scriptUrl,
+    country,
+    language,
+    tacocatEnv,
+  } = getTacocatEnv(env.name, locale);
+
+  loadScript(literalScriptUrl)
+    .catch(() => ({})) /* ignore if literals fail */
+    .then(() => loadScript(scriptUrl))
+    .then(() => {
+      runTacocat(tacocatEnv, country, language);
+      resolve(false);
+    })
+    .catch((error) => {
+      console.error('Failed to load tacocat', error);
+      resolve(true);
+    });
+});
+
+export const formatOfferDetails = (o) => {
```

Comment:
Is the only change from `o` the `price`?  Could you instead just add `.price` to `o`?

Reply:
> Is the only change from `o` the `price`?
It may seem so, but not exactly. The `o` object contains many other properties that are not needed to be displayed. The purpose of `formatOfferDetails` is to filter out the object, so renaming the function to `filterOfferDetails` to better reflect its purpose.